### PR TITLE
adding resources and update solution.js

### PR DESCRIPTION
**Pull request by jessecogollo**
adding resources in problem.md of all exercises and update solution.js file of aggregation exercise.

### CODE REVIEW

File: exercises/connect/problem.md [hunk 1]
```diff
@@ -11,7 +11,9 @@ mkdir data
 
 To start mongo on port 27017, run `mongod --port 27017 --dbpath=./data`.
 
-Then, in another terminal, run `learnyoumongo verify`.
+Then, in another terminal, run `npm install mongodb` if necessary then running with `sudo`.
```
Comment (evanlucas) [inline exercises/connect/problem.md]: Not sure if the mention of `sudo` should be included here.

File: exercises/find/problem.md [hunk 2]
```diff
@@ -26,6 +26,8 @@ To find a document or documents, one needs to call `find()` on the collection.
 
 Find is a little bit different than what we are used to seeing.
 
+take in mind, retrieve data from `process.argv` is an string, for convert data to integer you should use parseInt().
```
Comment (evanlucas) [inline exercises/find/problem.md]: Maybe reword to something like:

Keep in mind, `process.argv` is an array of strings. To convert to an integer, you could use `parseInt()`

**Comment by evanlucas** [comment]
Thanks for the contributions!!

**Comment by jessecogollo** [comment]
@evanlucas is ready your recommendation.

**Comment by evanlucas** [comment]
Thanks!
